tests/test_validation_pandapipes.py

```diff
@@ -0,0 +1,268 @@
+import os
+import sys
+from pathlib import Path
+from unittest import TestCase
+
+from mesido.constants import GRAVITATIONAL_CONSTANT
+from mesido.esdl.esdl_parser import ESDLFileParser
+from mesido.esdl.profile_parser import ProfileReaderFromFile
+from mesido.head_loss_class import HeadLossOption
+from mesido.util import run_esdl_mesido_optimization
+
+import numpy as np
+
+import pandapipes as pp
+from pandapipes.timeseries import run_timeseries
+
+import pandapower.control as control
+from pandapower.timeseries import DFData
+from pandapower.timeseries import OutputWriter
+
+import pandas as pd
+
+from utils_tests import demand_matching_test, energy_conservation_test, heat_to_discharge_test
+
+
+class ValidateWithPandaPipes(TestCase):
```

Comment:
This is an amazing test case.

To make it more complete I would suggest two minor efforts to make this more prominent.
- In the heat_physics.rst in docs add a paragraph that states that the headlosses are validated using pandapipes (ideally also include the graphs of the paper) and refer to this test case.
- In the README.md of our code base states: "with multiple linearization strategies to conservatively approximate the steady-state physics and financial models." I would suggest to add a sentence here: "See for example our head loss validation using pandapipes in test folder".

Reply:
Nice. README has been updated

Reply:
heat_physics.rst has also been updated